zone/achievement_manager.cpp

```
@@ -0,0 +1,1065 @@
+		);
+		const auto definition_index = FindDefinitionIndex(achievement_id);
+		if (!component_index || !definition_index) {
+			LogError(
+				"Enabled achievement criterion [{}] references missing component [{}/{}/{}]",
+				ParseUInt64(row[0]),
+				achievement_id,
+				component_type,
+				ParseUInt32(row[3])
+			);
+			Clear();
+			return false;
```

Cppcheck (2.21.0)

[style] The function 'GetMaxFaction' is never used.

(unusedFunction)